spec/data-types-and-values.html

```diff
@@ -16,28 +16,18 @@ <h1>MakeRecordValue ( _entries_ )</h1>
         <p>This abstract operation MakeRecordValue takes a _entries_ argument. It is used to specify the creation of new Record value types. It performs the following steps when called:</p>
         <emu-alg>
             1. Assert: _entries_ is a List.
-            1. Let _reversedEntries_ be a new List containing the same values as the list _entries_ where the values are ordered as an Array of the same values has been sorted using `Array.prototype.reverse`.
+            1. Reverse _entries_.

```

Comment:
this seems to be mutating the list; it'd be great to avoid that.

Reply:
What about something like:

1. Let _sortedEntries_ be _entries_ in reverse.

Reply:
I'll do the same for _sortedEntries_